learning_journal/routes.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+def includeme(config):
+    """Might include views."""
+    config.add_static_view('static', 'static', cache_max_age=3600)
+    config.add_route('home', '/')
+    config.add_route('detail', '/detail')
```

Comment:
Your detail route should take an id number, or some other unique identifier for individual journal entries